libnmstate/ifaces/ifaces.py

```diff
@@ -140,6 +141,24 @@ def _bring_slave_up_if_not_in_desire(self):
                         slave_iface.mark_as_up()
                         slave_iface.mark_as_changed()
 
+    def _validate_ovs_patch_peers(self):
+        """
+        When OVS patch peer does not exist or is down, raise an error.
+        """
+        for iface in self._ifaces.values():
+            if iface.is_patch_port:
```

Comment:
Only `OvsInternalIface` implemented this. You will get error:

AttributeError: 'EthernetIface' object has no attribute 'is_patch_port'

Reply:
You are right, fixed!